akka-persistence-query/src/main/scala/akka/persistence/query/typed/EventEnvelope.scala

```diff
@@ -81,6 +122,8 @@ object EventEnvelope {
  *
  * It is an improved `EventEnvelope` compared to [[akka.persistence.query.EventEnvelope]].
  *
+ * @param tags Some set with tags if journal does support extracting, None if not
```

Comment:
Do we need to distinguish between supporting or not in the api? This envelope is only used by eventsBySlices and we are in control of those implementations. Shouldn't be much harm to use empty Set for "unsupported"?

Reply:
I thought it'd be good for us to be able to detect not read vs no tags and throw in the filtering logic, not strictly necessary no.